ui/components/ProgressBar/ProgressBar.js

```diff
@@ -0,0 +1,174 @@
+import ProgressBarStyle from "./ProgressBarStyle";
+
+const ProgressDisplay = ({stepNumber}) => {
+    return (
+      <ul className="timeline" id="timeline">
+
+        <li className={stepNumber === 0 ? ("li active") : stepNumber > 0 ? ("li complete") : ("li")}>
+          <div className="status">
```

Comment:
The content to be shown inside this `div` should also be passed on as a prop.

Reply:
This div is for the circular fillings after each step is completed and does not have any content in it.

Reply:
yep. I get. But I am saying that it should be ideally be up to the consumer as to what should be displayed inside of the div...